road/validation_road.py

```diff
@@ -0,0 +1,113 @@
+import os
+import pandas as pd
+from simpledbf import Dbf5
+import configparser
+import json
+import yaml
+from sklearn.linear_model import LinearRegression
+from io import StringIO
+import geopandas as gpd
+from scatter import *
+
+
+config = configparser.ConfigParser()
+config.read('config.ini')
+
+# Extract the CHAMP input file names
+dbf_directory = config['DBF']['DBF_Directory']
+dbf_files = [dbf_file.strip() for dbf_file in config['DBF']['DBF_Files'].split(',')]
+dbf_column_names = [name.strip() for name in config['DBF']['DBF_Column_Names'].split(',')]
+
+# Extract the observed file name and tab
+excel_file_path = config['OBSERVED FILE']['Excel_File_Path']
+sheet_name = config['OBSERVED FILE']['Sheet_Name']
+extra_columns = [col.strip() for col in config['OBSERVED FILE']['Excel_Extra_Columns'].split(',')]
+obs_usecols = config['OBSERVED FILE']['Obs_usecols']
+time_periods = ['AM', 'MD', 'PM', 'EV', 'EA']
+
+#Part 1 

```

Comment:
I see you deleted out the "Part 1" line in the config file. Wondering if you wanted to delete that line here

Reply:
Oh I added it back on my end, the stats and geo config should be part2 and part 3